src/kendo.menu.js

```diff
@@ -462,6 +502,78 @@ var __meta__ = { // jshint ignore:line
             }
         },
 
+        _initOverflow: function(options) {
+            if (!options.overflow) {
+                return;
+            }
+
+            var that = this;
+            var isHorizontal = options.orientation == "horizontal";
+            var backwardBtn, forwardBtn;
+            that._openedPopups = {};
+            that._isRtl = kendo.support.isRtl(that.wrapper);
+            that._overflowWrapper = that.element.wrap("<div class='k-menu-scroll-wrapper " + options.orientation + "'></div>").parent();
+
+            if (isHorizontal) {
+                backwardBtn = $(templates.scrollButton({direction: "left"})).css({top: 0, left: 0, height: "100%"});

```

Comment:
We could probably add some well named classes as parameters and use CSS instead of adding the css with jQuery.